app/src/main/kotlin/com/wire/android/ui/authentication/create/details/CreateAccountDetailsScreen.kt

```diff
@@ -236,14 +234,21 @@ private fun DetailsContent(
                         labelMandatoryIcon = true,
                         descriptionText = stringResource(R.string.create_account_details_password_description),
                         imeAction = ImeAction.Next,
-                        autofillTypes = listOf(),
                         modifier = Modifier
                             .padding(horizontal = MaterialTheme.wireDimensions.spacing16x)
                             .testTag("password"),
                         state = if (state.error is CreateAccountDetailsViewState.DetailsError.TextFieldError.InvalidPasswordError) {
                             WireTextFieldState.Error()
                         } else {
                             WireTextFieldState.Default
+                        },
+                        autofill = false,
+                        onTap = {
+                            coroutineScope.launch {
+                                keyboardController?.show()
+                                focusRequesterTeamName.requestFocus()
+                                listState.animateScrollToItem(2)
```

Comment:
Maybe we can name what this 2 means in the context of this list ? :D

Reply:
you asked for a rename i remove the whole thing since it is not needed